.flowconfig

```diff
@@ -19,6 +19,7 @@
 ; "node_modules/react-native" but in the source repo it is in the root
 .*/Libraries/react-native/React.js
 .*/Libraries/react-native/ReactNative.js
+.*/Libraries/Renderer/src/renderers/shared/utils/ReactErrorUtils.js
```

Comment:
Have you cleared your cache on circle CI builds recently? After pulling in the latest set of commits, I started to get failures on the javascript flow tests related to that module being a duplicate. I wonder if circle CI is running with an older version of node modules for a certain dependency? If not, then it may be a difference in our docker environment which I can definitely dig into more.

Also, I will update this PR stripping out the `ScrollView` tests for instrumentation. I noticed they were removed from your circle CI test run suite.

@ericvicenti